### PR TITLE
chore: reduce PHPStan errors in PhpUnitAttributesFixer

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: src/Fixer/PhpUnit/PhpUnitAttributesFixer.php [hunk 1]
```diff
@@ -334,6 +337,7 @@ private static function fixDataProvider(Tokens $tokens, int $index, Annotation $
         }
 
         if (str_contains($matches[1], '::')) {
+            // @phpstan-ignore-next-line offsetAccess.notFound
```
Comment (Wirone) [inline src/Fixer/PhpUnit/PhpUnitAttributesFixer.php:340]:
```suggestion
            // @phpstan-ignore offsetAccess.notFound
```

`@phpstan-ignore` automatically detects if the error is in the same or in the next line. I don't want to spam with similar comments, but it applies to multiple other places too.

**Review by Wirone — APPROVED** [review]

**Review by kubawerlos — CHANGES_REQUESTED** [review]
Isn't that silly to add `if` conditions that will never be used just to make PHPStan bit happier?

File: src/Fixer/PhpUnit/PhpUnitAttributesFixer.php [hunk 2]
```diff
@@ -307,6 +307,9 @@ private static function fixWithEnabledDisabledValue(Tokens $tokens, int $index,
     private static function fixCovers(Tokens $tokens, int $index, Annotation $annotation): array
     {
         $matches = self::getMatches($annotation);
+        if (!isset($matches[1])) {
```
Comment (kubawerlos) [inline src/Fixer/PhpUnit/PhpUnitAttributesFixer.php:310]: Please add test for it or remove it. Thank you.

File: src/Fixer/PhpUnit/PhpUnitAttributesFixer.php [hunk 3]
```diff
@@ -307,6 +307,7 @@ private static function fixWithEnabledDisabledValue(Tokens $tokens, int $index,
     private static function fixCovers(Tokens $tokens, int $index, Annotation $annotation): array
     {
         $matches = self::getMatches($annotation);
+        assert(!isset($matches[1]));
```
Comment (kubawerlos) [inline src/Fixer/PhpUnit/PhpUnitAttributesFixer.php:310]:
```suggestion
        assert(isset($matches[1]));
```

File: src/Fixer/PhpUnit/PhpUnitAttributesFixer.php [hunk 4]
```diff
@@ -407,6 +409,7 @@ private static function fixDepends(Tokens $tokens, int $index, Annotation $annot
     private static function fixRequires(Tokens $tokens, int $index, Annotation $annotation): array
     {
         $matches = self::getMatches($annotation);
+        assert(!isset($matches[1]));
```
Comment (kubawerlos) [inline src/Fixer/PhpUnit/PhpUnitAttributesFixer.php:412]:
```suggestion
        assert(isset($matches[1]));
```

Comment (keradus) [reply src/Fixer/PhpUnit/PhpUnitAttributesFixer.php]: (noticed too, just pushed)

**Review by kubawerlos — APPROVED** [review]

**Pull request merged**